CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+# v2.3.0
+
+This release migrates to the new `osv.dev` and `osv-schema` proto bindings for its internal data models ([#2328](https://github.com/google/osv-scanner/pull/2328)). This is primarily an internal change and should not impact users.
+
+### Features:
+
+- [Feature #2321](https://github.com/google/osv-scanner/pull/2321) Add support for license checks for RubyGems.
+- [Feature #2294](https://github.com/google/osv-scanner/pull/2294) Replace `requirementsenhanceable` extractor with transitive enricher.
+- [Feature #1417](https://github.com/google/osv-scanner/issues/1417) Add `packagedeprecation` enricher to check for deprecated packages. (`--experimental-plugins=packagedeprecation`)
+- [Feature #1436 (osv-scalibr)](https://github.com/google/osv-scalibr/pull/1436) Add support for `mysql_config_editor` configuration files. (`--experimental-plugins=secrets/mysqlmylogin`)
+- [Feature #1509 (osv-scalibr)](https://github.com/google/osv-scalibr/pull/1509) Add VAPID keys detector. (`--experimental-plugins=secrets/vapidkey`)
+- [Feature #1498 (osv-scalibr)](https://github.com/google/osv-scalibr/pull/1498) Add Cloud Storage HMAC keys validator. (`--experimental-plugins=secrets/gcshmackey`)
```

Comment:
Hmm I think let's leave these out for now, I think secret scanning only really works in JSON output format, as we don't really support showing findings yet in the other formats.